README.md

```diff
@@ -0,0 +1,342 @@
+    types:
+    - *
+  workerTemplate:
+    defaultConfigFiles:
+      brigade.js: |-
+        const { events } = require("@brigadecore/brigadier");
+
+        events.on("brigade.sh/slack", "demo", () => {
+          console.log("Someone sent the /demo slash command!");
+        });
+
+        events.process();
+```
+
+In the alternative example below, we subscribe _only_ to `demo` events:
+
+```yaml
+apiVersion: brigade.sh/v2-beta
+kind: Project
+metadata:
+  id: slack-demo
+description: A project that demonstrates integration with Slack
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/slack
+    qualifiers:
+      appID: FAKEAPPID
+    types:
+    - demo
+  workerTemplate:
+    defaultConfigFiles:
+      brigade.js: |-
+        const { events } = require("@brigadecore/brigadier");
+
+        events.on("brigade.sh/slack", "demo", () => {
+          console.log("Someone sent the /demo slash command!");
+        });
+
+        events.process();
+```
+
+Assuming this file were named `project.yaml`, you can create the project like
+so:
+
+```console
+$ brig project create --file project.yaml
+```
+
+Typing `/demo` in a Slack channel should now send an event (slash command) from
+Slack to your gateway. The gateway, in turn, will emit the event into Brigade's
+event bus. Brigade should initialize a worker (containerized event handler) for
+every project that has subscribed to the event, and the worker should execute
+the `brigade.js` script that was embedded in the project definition.
+
+List the events for the `slack-demo` project to confirm this:
+
+```console
+$ brig event list --project slack-demo
+```
+
+Full coverage of `brig` commands is beyond the scope of this documentation, but
+at this point, additional `brig` commands can be applied to monitor the event's
+status and view logs produced in the course of handling the event.
+
+## Events Received and Emitted by this Gateway
+
+Unlike most Brigade gateways, this gateway dynamically determines the value of
+the `type` field on events it emits into Brigade's event bus by simply stripping
+the leading slash (`/`) from the incoming slash command.
+
+All events are qualified by `appID` and labeled with `teamID` (workspace ID),
+`channelID`, and `userID`. Qualifying events by `appID` ensures disambiguation
+if you are using multiple Slack Apps that each support one or more slash
+commands having the same names. Labeling event with `teamID`, `channelID`, and
+`userID` enables subscriptions to optionally be very specific about what events
+they're interested in.
+
+Here is an abbreviated representation of a sample event emitted by this gateway:
+
+```yaml
+apiVersion: brigade.sh/v2-beta
+kind: Event
+metadata:
+  created: "2021-09-24T20:46:47.037Z"
+  id: 33b0e475-1414-4142-a682-f7b916ad2989
+projectID: slack-demo
+source: brigade.sh/slack
+qualifiers:
+  appID: A02C3GPHM6D
+labels:
+  channelID: C02CU7ZMJMA
+  teamID: TFNLDR3SP
+  userID: UGN3UHXAM
+type: demo
+payload: foobar
```

Comment:
What would the payload be for this gateway?  Anything after the slash command?  In this case, `/demo foobar`?

Regardless of if I have that right, should we add a mention around what the payload represents here (and how to set it)?

Reply:
Yes. I totally failed to document where the payload comes from, but you guessed correctly. Will fix.

Reply:
I can't tell if this has been addressed in the latest changes.

Reply:
Nope. Not yet.

Reply:
Fixed. The change is above this sample event, so it doesn't show as "outdated" above.